mitodl/ocw-hugo-themes
# instructors from static api

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/177

#### What's this PR do?
In https://github.com/mitodl/ocw-to-hugo/pull/341, we changed the way `ocw-to-hugo` outputs instructor data.  The `instructors` key now contains just a `content` key with an array of UUID's of instructors in the same way `ocw-studio` outputs instructors linked to a course with the relation widget.  This PR updates the referenced version of `ocw-to-hugo` in `package.json` and updates the templates to fetch the instructors based on a configured `STATIC_API_BASE_URL` in the environment.

#### How should this be manually tested?
 - Read the readme if you have never spun up a course site locally before and make sure you have S3 access set up
 - Make `OCW_TEST_COURSE` is set to a valid course slug like `18-06sc-linear-algebra-fall-2011` and `STATIC_API_BASE_URL` is set to http://ocwnext.odl.mit.edu/
 - Spin up your course with `npm run start:course`
 - Visit http://localhost:3000 and make sure the course instructors are visibile within the course info section and that they have valid search urls associated with them
